.github/workflows/build-images-action.yml

```diff
@@ -8,49 +8,24 @@ on:
     tags:
     - 'v*'
 
-permissions: {}
-
 jobs:
-  build:
-    name: Build container images
-    runs-on: ubuntu-latest
-
-    permissions:
```

Comment:
And if you need permissions, put this back as well.

Reply:
That wouldn't work either, the job was supposed to be taken as a whole, not as a step in the bigger wf:

```
The workflow is not valid. .github/workflows/build-images-action.yml (Line: 21, Col: 5): Unexpected value 'uses' .github/workflows/build-images-action.yml (Line: 22, Col: 5): Unexpected value 'with'
```

https://github.com/metal3-io/baremetal-operator/actions/runs/10300436232

Reply:
OK, let's go with global read first, and iterate from that if it is improvable.